Spinner/Spinner.js

```diff
@@ -57,17 +63,29 @@ const SpinnerCore = kind({
 	},
 
 	computed: {
+		'aria-label': ({'aria-label': label, type}) => {
+			// TODO: These static values will need to be localized
+			return label || type === 'searching' ? 'Searching' : 'Loading';
+		},
 		className: ({styler, type}) => styler.append(type),
-		spinnerNodes: ({css, type}) => type === 'searching' ? createSpinnerNodes(12, css) : createSpinnerNodes(4, css)
+		spinnerNodes: ({styler, type}) => {
+			return Array.from({length: type === 'searching' ? 12 : 4}, (_, index) => (
+				<span className={styler.join('fan', `fan${index + 1}`)} key={`fan${index}`} />
+			));
+		}
 	},
 
-	render: ({css, spinnerNodes, type, ...rest}) => (
-		<div aria-label={type} aria-live="off" role="alert" {...rest}>
-			<div className={css.bg}>
-				{spinnerNodes}
+	render: ({css, spinnerNodes, ...rest}) => {
+		delete rest.type;
+
+		return (
```

Comment:
Although it is more terse to "return" the JSX from the arrow fn, we usually have to change it later to remove props so it's often better to do so initially so the later diffs are more clear.